aequilibrae/matrix/aequilibrae_matrix.py

```diff
@@ -293,41 +326,57 @@ def robust_name(input_name: str, max_length: int, forbiden_names: List[str]) ->
 
         # copy all indices
         if avail_idx:
-            for nidx, idx in zip(idx_names, do_idx):
+            for nidx, idx in enumerate(do_idx):
                 ix = np.array(list(src.mapping(idx).keys()))
-                self.indices[nidx][:] = ix[:]
+                self.indices[:, nidx] = ix[:]
         else:
-            self.index[:] = np.arange(zones)
+            self.index[:, 0] = np.arange(zones)
 
         self.indices.flush()
 
-    def __load__(self):
+    def __load_aem__(self):
         # GET File version
-        self.__version__ = np.memmap(self.file_path, dtype="uint8", offset=0, mode="r+", shape=1)[0]
+        self.__version__ = np.memmap(
```

Comment:
I'd clean this up so that it all fits on one line and makes it easy to see the overall flow of the binary format

Reply:
Done